android/src/main/java/com/thebluealliance/androidclient/adapters/ViewMatchFragmentPagerAdapter.java

```diff
@@ -0,0 +1,49 @@
+package com.thebluealliance.androidclient.adapters;
+
+import com.thebluealliance.androidclient.fragments.match.MatchBreakdownFragment;
+import com.thebluealliance.androidclient.fragments.match.MatchInfoFragment;
+
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+public class ViewMatchFragmentPagerAdapter extends FragmentPagerAdapter {
+
+    public final String[] TITLES = {"Results", "Breakdown"};
```

Comment:
I've been meaning to move the viewpager titles to string resources; want to start that here?

I'd switch this array to be an array of string resource ints and pass a Context to the constructor. There, just populate a second array of strings with the resolves strings from the context.